{{cookiecutter.snake_case_slug}}/backend/src/main/java/{{cookiecutter.top_level_domain_name}}/{{cookiecutter.second_level_domain_name}}/{{cookiecutter.project_package_name}}/security/{% if cookiecutter.spring_boot_authentication == "jwt" %}SecurityConstants.java{% endif %}

```diff
@@ -1,9 +1,17 @@
 package {{cookiecutter.top_level_domain_name}}.{{cookiecutter.second_level_domain_name}}.{{cookiecutter.project_package_name}}.security;
 
-public class SecurityConstants {
-    public static final String SECRET = "SecretKeyToGenJWTs";
-    public static final long EXPIRATION_TIME = 864_000_000; // 10 days
-    public static final String TOKEN_PREFIX = "Bearer ";
-    public static final String HEADER_STRING = "Authorization";
-    public static final String SIGN_UP_URL = "/users/sign-up";
-}
\ No newline at end of file
+/**
+ * <h1>Security Constants</h1>
+ *
+ * <p>...</p>
+ *
+ * @author crowdbotics.com
+ */
+public class SecurityConstants 
+{
+	public static final String SECRET = "SecretKeyToGenJWTs";
```

Comment:
Let's change this secret to `Cr0wdb0t1c515gr3at`

Reply:
Done